test/package.json

```diff
@@ -10,19 +10,19 @@
   },
   "author": "",
   "dependencies": {
-    "bedrock": "^3.1.1",
-    "bedrock-account": "^4.1.0",
+    "bedrock": "^4.1.1",
+    "bedrock-account": "^5.0.0",
     "bedrock-did-context": "^1.0.0",
     "bedrock-express": "^3.2.0",
     "bedrock-https-agent": "^2.0.0",
     "bedrock-identity": "^8.0.0",
```

Comment:
Is bedrock-identity needed here? If not, please remove.

Reply:
It does not appear to be used, so, I removed it.